### ISSUE TITLE
`yarn build` errors should be listed as _flaws_

### ISSUE THREAD

**Issue by teoli2003** [mdn/yari#5285]
When running `yarn build`, I notice that errors are detected and logged.

For example:
![Screenshot 2022-02-15 at 08 55 36](https://user-images.githubusercontent.com/1466293/154017328-2b7be093-b2ed-4179-b552-09aa77c1a41b.png)

These errors go unnoticed.

One possible solution would be to transform these into _flaws_ associated with the article. They'll appear 1) in the content ci  2) at the top of the page in the flaw list  3) in the flaw dashboard.

As we are working to lower the number of flaws (little by little), they will eventually get fixed (unlike now as they are unnoticed).

cc/ @ddbeck

**Comment by ddbeck** [comment]
I agree that these errors should be promoted a bit, so that they're not so easily ignored. That said, I do not think we should invest any more effort in the flaws system. If I had my way, we'd have a common, low-level way of reporting build issues, such as some JSON output from Yari which could be used to generate reports, block merges, build a dashboard, etc. Neither the existing flaws system nor the build error messages are suitable for any of these applications.

If these specific errors are getting lost, I think it would be a great place to begin creating a post-flaws system. As a very rough set of requirements, I'd like to see:

- Output to JSON
- In a structure similar to the content `inventory` command (i.e., grouped by file, with objects or arrays of metadata and errors)
- A structured error object with an error name/code, human-readable description, etc.

(Imagine I went on a rant here about the demoralizing impact of the flaws system and how I don't want to recapitulate it.)

**Comment by teoli2003** [comment]
I would be very happy to have another system than flaws :-) A JSON would be perfect for me.

**Comment by caugner** [comment]
This issue is obsolete with our new [rari](https://github.com/mdn/rari) build system.
